bin/offline-secrets.sh

```diff
@@ -31,6 +31,9 @@ brig:
       secret: "$zrest"
     awsKeyId: dummykey
     awsSecretKey: dummysecret
+    rabbitmq:
```

Comment:
no hard coded secrets please. Yes, randomly generating them and handling them everywhere is sometimes hard. :)